src/pet.js

```diff
@@ -69,6 +69,11 @@ function Pet(pname) {
         }
     },
 
+    haveBaby(){
```

Comment:
do you want to put a parameter here so the user an input their own name?

Reply:
Oh it was supposed to, I forgot! Thanks for pointing it out.